indico/modules/events/registration/schemas.py

```diff
@@ -86,6 +89,12 @@ def _get_filenames(self, registration):
                 for r in registration.data
                 if r.field_data.field.field_impl.is_file_field and r.storage_file_id is not None}
 
+    def _get_personal_data_picture(self, registration):
+        picture = registration.get_personal_data_picture()
+        if picture is None:
+            return
+        return url_for('.registration_picture', picture.locator.file, _external=True)
```

Comment:
I'm wondering if this endpoint is accessible with the `registrants` oauth scope (that's what the app has). The `everything` scopes should work for any endpoint but not sure about `registrants`. If not, we might need to add `@oauth_scope('registrants')` to the RH.

Reply:
I suggested adding another RH for it as part of the check-in APIs yesterday... feels cleaners to me